Clean/Gadgets/Addition8/Addition8FullCarry.lean

```diff
@@ -134,7 +134,7 @@ def lookupCircuit : LookupCircuit (F p) Inputs Outputs := {
   name := "Addition8FullCarry"
 
   computableWitnesses n input := by
-    simp_all only [circuit_norm, circuit, main,
+    simp_all only [circuit_norm, circuit, main, FormalAssertion.toSubcircuit,
       Operations.forAllFlat, Operations.toFlat, FlatOperation.forAll, Inputs.mk.injEq]
```

Comment:
this is still a very nice proof :D

Reply:
The same thing is needed for the same reason in MultiAND.

https://github.com/Verified-zkEVM/clean/pull/220/commits/925042b8c1ff628534c957dc29f3354dcfd48a0d

The thing is not that complicated but it's there. Probably need a lemma for handling this.